clients/models.py

```diff
@@ -4,20 +4,32 @@
 class Client(models.Model):
     name = models.CharField(max_length=100)
     address = models.CharField(max_length=100)
+    is_active = models.BooleanField(default=True)
 
     def __str__(self):
         return self.name
 
+    def save(self, *args, **kwargs):
```

Comment:
Please comment this function to explain why we are doing this. I understand because you told me, but in a while it might seem crpytic.